dashboard/lib/policies/lti.rb

```diff
@@ -117,4 +117,17 @@ def self.show_email_input?(user)
   def self.roster_sync_enabled?(user)
     user.teacher? && user.lti_roster_sync_enabled
   end
+
+  def self.early_access?
+    DCDO.get('lti_early_access_limit', false).present?
+  end
+
+  def self.early_access_closed?
+    return unless early_access?
```

Comment:
Is it better to consistently return a boolean, i.e. return `false` here? Or was this intentional to distinguish between the limit being reached (false) vs. early access not being enabled at all (nil)? Non-blocker, just curious

Reply:
Correct, it was done intentionally. If early access isn't enabled, then logically, it can't be closed.
Since `nil` is also a falsy value, it doesn't compromise the method's purpose but rather provides an extra indicator.